src/Behat/Testwork/Tester/Result/TestResult.php

```diff
@@ -23,6 +23,8 @@ interface TestResult
     public const UNDEFINED = 30;
     public const FAILED = 99;
 
+    public const NO_TESTS = -100;
```

Comment:
We move this constant to this class so that it can be part of the constants associated to the TestResult class

Reply:
I'm not sure this is the right solution. Places dealing with the result of a single test would never need to care about the `NO_TESTS` result.